server/twitch.handler.ts

```diff
@@ -71,9 +69,8 @@ export class TwitchHandler {
       });
     });
 
-
     this.tmiClient.on('cheer', (channel, tags, message) => {
-      const command = this.getCommandOfMessage(message);
+      const command = this.getCommandOfMessage(message, { event: TwitchEventTypes.bits, bitsCount: parseInt(tags.bits) });
```

Comment:
maybe just push `tags` with event-type to `getCommandOfMessage` for upcoming checks tags will be probably needed anyway